docs/user_guide/selection/DropHighPSIFeatures.rst

```diff
@@ -68,6 +68,21 @@ to the PSI value. The most commonly used thresholds are:
 - Below 10%, the variable has not experienced a significant shift.
 - Above 25%, the variable has experienced a major shift.
 - Between those two values, the shift is intermediate.
+- 'auto' threshold will be calculated based on size of dataset and number of bins.
+
+For 'auto' formula we are using chi2 approximation, proposed by B. Yurdakul:
```

Comment:
Could we modify slightly to:

When 'auto', the threshold is calculated using the chi2 approximation, proposed by B. Yurdakul: